includes/model/class-user.php

```diff
@@ -97,6 +97,17 @@ public function get_name() {
 		return \esc_attr( \get_the_author_meta( 'display_name', $this->_id ) );
 	}
 
+	/**
+	 * Set the User-Name.
+	 *
+	 * @param string $name The User-Name.
+	 * @return bool       True if the User-Name was updated, false otherwise.
+	 */
+	public function save_name( $name ) {
```

Comment:
I would prefer to move the save functions to the EMA-API class instead then, because they are very specific and I do not know if we can re-use them in other parts of the code.

Reply:
For the User type I completely agree, as long as we and EMA are both using the same attributes